scripts/globals/abilities/high_jump.lua

```diff
@@ -52,7 +52,7 @@ function onUseAbility(player,target,ability,action)
     if (tpHits + extraHits > 0) then
         -- Under Spirit Surge, High Jump reduces TP of target
         if (player:hasStatusEffect(EFFECT_SPIRIT_SURGE) == true) then
-            target:delTP(damage * 2)
+            target:delTP((damage * 2) / 10)
```

Comment:
Ugh, you're right.  I did that in other places.  I guess I was tired when I did that one.